src/containers/AppBody.jsx

```diff
@@ -112,8 +112,8 @@ export class AppBody extends Component {
 
         const permissions = {
           canRead: true,
-          // Currently, users may only edit a file they own
-          canEdit: (username === cookie.load('username')),
+          canEdit: (username === cookie.load('username') ||
```

Comment:
I already have refactored that in [this PR](https://github.com/maryvilledev/codesplainUI/pull/464)